R/ihw_class.R

```diff
@@ -47,13 +51,17 @@ ihwResult <- setClass("ihwResult",
            		     alpha = "numeric",
                    nbins = "integer",
                    nfolds = "integer",
-                   regularization_term = "numeric",
+                   regularization_term = "matrix",
                    m_groups = "integer",
                    penalty = "character",
                    covariate_type = "character",
                    adjustment_type = "character",
                    reg_path_information = "data.frame",
-           		     solver_information= "list"))
+           		     solver_information= "list",
+           		     stratification_method = "character",
+           		     weight_matrices_forest = "list",
+           		     ntaus = "integer",
```

Comment:
I would avoid having greek letter variable names in public-facing methods or objects. They only make sense if you know which formula the letter refers to. I would thus recommend renaming all occurrences of `ntaus` to some more literal variable name like `n_thresholds` or `n_censor_thres`

Reply:
I changed all `ntaus` to `n_censor_thres` in the whole project